zerver/lib/alert_words.py

```diff
@@ -13,6 +15,21 @@ def alert_words_in_realm(realm: Realm) -> Dict[int, List[str]]:
     user_ids_with_words = dict((user_id, w) for (user_id, w) in all_user_words.items() if len(w))
     return user_ids_with_words
 
+@cache_with_key(realm_alert_words_automaton_cache_key, timeout=3600*24)
+def get_alert_word_automaton(realm: Realm) -> ahocorasick.Automaton:
+    user_id_with_words = alert_words_in_realm(realm)
+    alert_word_automaton  = ahocorasick.Automaton()
+    for (user_id, alert_words) in user_id_with_words.items():
+        for alert_word in alert_words:
+            trimmed_alert_word = alert_word.lower()
```

Comment:
why is this called `trimmed_alert_word`?  It looks like it's just a lower-case version.  Would be better to call it `alert_word_lower` or something similar.